datasets/new_york_taxi_trips/pipelines/dataset.yaml

```diff
@@ -0,0 +1,30 @@
+# Copyright 2021 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+dataset:
+  name: :new_york_taxi_trips
+  friendly_name: new_york_taxi_trips
+  description: "New York Taxi Trips dataset"
+  dataset_sources: ~
+  terms_of_use: ~
+
+
+resources:
+  - type: bigquery_dataset
+    dataset_id: new_york_taxi_trips
+    description: "TLC Green trips 2018 dataset"
+  - type: storage_bucket
+    name: new-york-taxi-trips
+    uniform_bucket_level_access: True
+    location: US
```

Comment:
We can remove this and regenerate the TF files

Reply:
Done